rust/runtime/tests/test_tvm_static/src/main.rs

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+extern crate ndarray;
+#[macro_use]
+extern crate tvm_runtime;
+
+use ndarray::Array;
+use tvm_runtime::DLTensor;
+
+mod tvm_mod {
+    import_module!("runtime/tests/test_tvm_static/test.o");
```

Comment:
Is there a way to have this generate better wrappers for the contained packed functions, instead of having to use macros to call them?

Reply:
What did you have in mind? The only other alternative would be to have `build.rs` dump the macro output to a file (or write a compiler plugin, but that's the nuclear option).